ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -445,7 +445,7 @@ export default function ViewQuote() {
           : null,
         initialGasPrice: gasPrice,
         initialGasLimit: maxGasLimit,
-        minimumGasLimit: nonCustomMaxGasLimit,
+        minimumGasLimit: parseInt(nonCustomMaxGasLimit, 16),
```

Comment:
Fixes the issue with the minimum gas limit error message showing a hex number

Reply:
Couldn't `parseInt` be unsafe here?
```suggestion
        minimumGasLimit: new BigNumber(nonCustomMaxGasLimit, 16).toString(10),
```

Reply:
`nonCustomMaxGasLimit` question addressed here https://github.com/MetaMask/metamask-extension/pull/9858/files#r521741706